app/src/main/java/com/hapramp/steem/PermlinkGenerator.java

```diff
@@ -13,8 +13,18 @@
   private static final int MAX_LENGTH = 64;
   private final static String ALPHA_NUMERIC_STRING = "q0w1e2r3t4y5u6i7o8p9l0kjhgfdsazxcvbnm";
 
-  public static String getPermlink() {
+  public static String getPermlink(String title){
+    title = title.trim().toLowerCase();
+    if (title.length() > 0) {
+      title = title.replaceAll("[ ]{1,}", "-");
```

Comment:
What happens if there's a `/` in the permlink text?

I suggest replacing everyting non alphanumeric with `-`.